store/sqlstore/user_store.go

```diff
@@ -707,65 +669,58 @@ type UserWithLastActivityAt struct {
 
 func (us SqlUserStore) GetRecentlyActiveUsersForTeam(teamId string, offset, limit int) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		var users []*UserWithLastActivityAt
+		query := usersQuery.
+			Select("s.LastActivityAt").
+			Join("TeamMembers tm ON (tm.UserId = u.Id AND tm.TeamId = :TeamId)").Bind("TeamId", teamId).
+			Join("Status s ON (s.UserId = u.Id)").
+			OrderBy("s.LastActivityAt DESC").
+			OrderBy("u.Username ASC").
+			Offset(offset).Limit(limit)
 
-		if _, err := us.GetReplica().Select(&users, `
-            SELECT
-                u.*,
-                s.LastActivityAt
-            FROM Users AS u
-                INNER JOIN TeamMembers AS t ON u.Id = t.UserId
-                INNER JOIN Status AS s ON s.UserId = t.UserId
-            WHERE t.TeamId = :TeamId
-            ORDER BY s.LastActivityAt DESC
-            LIMIT :Limit OFFSET :Offset
-            `, map[string]interface{}{"TeamId": teamId, "Offset": offset, "Limit": limit}); err != nil {
+		var users []*UserWithLastActivityAt
+		if _, err := us.GetReplica().Select(&users, query.String(), query.Args()); err != nil {
 			result.Err = model.NewAppError("SqlUserStore.GetRecentlyActiveUsers", "store.sql_user.get_recently_active_users.app_error", nil, err.Error(), http.StatusInternalServerError)
-		} else {
-
-			userList := []*model.User{}
+			return
+		}
 
-			for _, userWithLastActivityAt := range users {
-				u := userWithLastActivityAt.User
-				u.Sanitize(map[string]bool{})
-				u.LastActivityAt = userWithLastActivityAt.LastActivityAt
-				userList = append(userList, &u)
-			}
+		userList := []*model.User{}
 
-			result.Data = userList
+		for _, userWithLastActivityAt := range users {
+			u := userWithLastActivityAt.User
+			u.Sanitize(map[string]bool{})
+			u.LastActivityAt = userWithLastActivityAt.LastActivityAt
+			userList = append(userList, &u)
 		}
+
+		result.Data = userList
 	})
 }
 
 func (us SqlUserStore) GetNewUsersForTeam(teamId string, offset, limit int) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		var users []*model.User
+		query := usersQuery.
+			Join("TeamMembers tm ON (tm.UserId = u.Id AND tm.TeamId = :TeamId)").Bind("TeamId", teamId).
+			OrderBy("u.CreateAt DESC").
+			OrderBy("u.Username ASC").
```

Comment:
Adding an explicit ordering here in addition to `u.CreateAt` to simplify unit testing and remove non-determinism, as above.